src/components/listings/AlbumListing.tsx

```diff
@@ -0,0 +1,58 @@
+import { format } from 'date-fns'
+import React from 'react'
+import { View, Text, Image, ImageBackground } from 'react-native'
+
+interface AlbumListingProps {
+  title: string
+  dateFrom: Date
+  dateTo: Date
+  images?: Array<typeof Image>
```

Comment:
Can we use `T[]` instead of `Array<T>` please :)

Reply:
👍